app/components/card.js

```diff
@@ -7,6 +11,20 @@ const Card = ({ url }) => {
     url.final_url == "Error" ||
     url.final_url == "Not found"
   );
+  /** Code Feedback - great use of using intermediate variables
```

Comment:
Also, your casing changes from camelCase to snake_case. I think some of your python was sneaking in. 

Try to keep the casing consistent within the project and consistent with the language norm.

Reply:
Woops, definitely just forgot what language I was working in. Good catch.